qemu/tests/cfg/blockdev_mirror_simple.cfg

```diff
@@ -0,0 +1,71 @@
+# Storage backends:
+#   filesystem, iscsi_direct, ceph, nbd, gluster_direct
+# The following testing scenario is covered:
+#   Block mirror granularity test
+#     The mirror image is a local image(filesystem)
+#     granularity is 512, 2M and a random value from (512, 2M)
+#   Block mirror buf-size test
+#     The mirror image is a local image(filesystem)
+#     buf-size = count * granularity, when
+#     count is a random value from [1, 10], while
+#     granularity is a random value from (512, 2M)
+
+- blockdev_mirror_simple:
+    only Linux
+    only filesystem iscsi_direct ceph nbd gluster_direct
+    start_vm = no
+    qemu_force_use_drive_expression = no
+    type = blockdev_mirror_simple
+    virt_test_type = qemu
+    images += " data1"
+    source_images = data1
+    target_images = mirror1
+    remove_image_data1 = yes
+    force_create_image_data1 = yes
+    backup_options_data1 = sync granularity
+    sync = full
+    storage_pools = default
+    storage_pool = default
+
+    image_size_data1 = 2G
+    image_size_mirror1 = ${image_size_data1}
+
+    image_format_data1 = qcow2
+    image_format_mirror1 = qcow2
+
+    image_name_data1 = data1
+    image_name_mirror1 = mirror1
+    rebase_mode = unsafe
+
+    nbd:
+        nbd_port_data1 = 10822
+        force_create_image_data1 = no
+        image_create_support_data1 = no
+    iscsi_direct:
+        lun_data1 = 1
+
+    # For local mirror images
+    storage_type_default = directory
+    enable_iscsi_mirror1 = no
+    enable_ceph_mirror1 = no
+    enable_gluster_mirror1 = no
+    enable_nbd_mirror1 = no
+    image_raw_device_mirror1 = no
+
+    variants:
+        - granularity_512:
+            granularity = 512
+            tempfile_size = 10M
+        - granularity_64M:
+            granularity = 64M
+            tempfile_size = 1000M
+        - random_betwen_512_64M:
+            granularity = random
+            granularity_list = 1K 2K 4K 8K 16K 32K 64K 128K 256K 512K 1M 2M 4M 8M 16M 32M
+
+            variants:
+                - buf_size_default:
+                - buf_size_random:
+                    buf-size = random
+                    count_list = 1 2 3 4 5 6 7 8 9 10
```

Comment:
If we want to test the none default buf-size, then we don't need the 1.

Reply:
updated, thanks.